tests/models/test_patch_predictor.py

```diff
@@ -871,7 +871,7 @@ def test_command_line_models_file_not_found(sample_svs, tmp_path):
             "--file_types",
             '"*.ndpi, *.svs"',
             "--output_path",
-            tmp_path,
+            str(tmp_path.joinpath("output")),
```

Comment:
We should consider adding to our internal style guide a standardisation on path construction, e.g. these are all equivalent:

```python
tmp_path.joinpath("output")
tmp_path / "output"
Path(tmp_path, "output")
```

Only a very minor thing, but some consistency would be nice.

Reply:
Let's discuss this and decide on one style.
We can put it on OneNote and create a PR for consistency.